src/coreclr/jit/assertionprop.cpp

```diff
@@ -5260,7 +5260,7 @@ class AssertionPropFlowCallback
     {
         ASSERT_TP pAssertionOut;
 
-        if (predBlock->bbJumpKind == BBJ_COND && (predBlock->bbJumpDest == block))
+        if (predBlock->getBBJumpKind() == BBJ_COND && (predBlock->bbJumpDest == block))
```

Comment:
Looks like most uses can be replaced with an existing `block->KindIs(...)` (it also supports 'params').

Also, should it be Get or get?

Reply:
Thank you for catching that -- I'll push a revision converting all the bbJumpKind comparisons to KindIs().

I'm not sure what the case should be. Some BasicBlock methods (like `KindIs`) use Pascal case, while others (like `makeBlockHot`) use camel case.

Reply:
The convention for _all_ new code in the Jit is to use PascalCase, except in LSRA (where camelCase is used consistently).